.editorconfig

```diff
@@ -0,0 +1,17 @@
+root = true
+
+[*]
+insert_final_newline = true
+
+[*.{kt,kts}]
+max_line_length = 140
+indent_size = 4
+ij_kotlin_allow_trailing_comma = true
+ij_kotlin_allow_trailing_comma_on_call_site = true
+
+[*.java]
+ij_java_class_count_to_use_import_on_demand = 3
+ij_java_names_count_to_use_import_on_demand = 3
+ij_java_packages_to_use_import_on_demand = 3
+ij_java_blank_lines_after_class_header = 1
+ij_java_imports_layout = *,|,javax.**,java.**,|,$*
```

Comment:
Ah, I just realized this project has codestyle file committed, but my IDE still used the _default_ one :/ 
I thought there is no _suggested_ code style so I thought I should add one, to avoid unnecessary changes in the imports. 

I can remove the file, although I'd strongly suggest slowly migrating towards IDE-agnostic `.editorconfig`, and leave at least the upper part, up to kotlin codestyle (which I believe will be needed if you decide to introduce static code analysis).

Reply:
This is neat. I actually wasn't aware that there was an IDE agnostic config like this. Will this get other people's IDEs to have the right settings even if the scheme is set to default?

<img width="331" alt="CleanShot 2023-06-14 at 14 45 45@2x" src="https://github.com/airbnb/lottie-android/assets/1307745/d1038e64-b20e-448f-a8eb-169fc40e3aa6">

Reply:
>Will this get other people's IDEs to have the right settings even if the scheme is set to default?

Yes, the `.editorconfig` preferences have precedence over IDE settings by default (but can be explicitly disabled). Intellij-based IDEs display additional information: 

<img width="411" alt="image" src="https://github.com/airbnb/lottie-android/assets/36954793/e36cc161-c277-4c0d-80c9-224d48910e87">

>IDE agnostic config

I realized I already lied, partially. It's not 100% IDE agnostic - most of the properties start with `ij_`, which work only in IntelliJ based IDEs, but other properties like `indent_size` or `max_line_length` or `insert_final_newline` work in every popular IDE